tests/test_misclassifications.py

```diff
@@ -0,0 +1,106 @@
+from presc.misclassifications.misclassification_visuals import *
+from presc.misclassifications.misclass_rate import *
```

Comment:
Please import the individual functions you will use, even if it's a long list. This creates confusion, since in this case it's also importing `pd` for pandas, and the tests are running without error even though `pandas` is not imported here, so it's hiding the fact that this module also requires pandas.